src/cmd/ping.rs

```diff
@@ -51,7 +52,13 @@ impl Ping {
     ///
     /// The response is written to `dst`. This is called by the server in order
     /// to execute a received command.
-    #[instrument(skip(self, dst))]
+    #[instrument(
+        name = "Ping::apply",
+        skip(self, dst),
+        fields(
+            ?msg = self.msg,
```

Comment:
assuming the intent here is to use `?` syntax to record `msg` with debug, this should be
```suggestion
            msg = ?self.msg,
```